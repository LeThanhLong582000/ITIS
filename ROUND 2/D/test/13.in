3
11
25
21
