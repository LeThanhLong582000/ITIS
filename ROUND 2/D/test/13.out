773
7555935
547337
